packages/specs/schema/src/components/default/inheritedClassMapper.ts

```
@@ -4,6 +4,23 @@
 import {execMapper, registerJsonSchemaMapper} from "../../registries/JsonSchemaMapperContainer.js";
 import {getInheritedStores} from "../../utils/getInheritedStores.js";
 
+function alterMerge(_: string, obj: any) {
+  if (obj?.type && obj?.$ref) {
+    const {$ref, ...schema} = obj;
+
+    obj = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)